pkg/libovsdb/notation.go

```diff
@@ -149,17 +149,27 @@ func NewTransact(params []interface{}) (*Transact, error) {
 
 // TransactResponse represents the response to a Transact Operation
 type TransactResponse struct {
-	Result []OperationResult `json:"result"`
-	Error  string            `json:"error"`
+	Result []*OperationResult `json:"result"`
+	Error  string             `json:"error"`
 }
 
 // OperationResult is the result of an Operation
 type OperationResult struct {
-	Count   int         `json:"count,omitempty"`
-	Error   string      `json:"error,omitempty"`
-	Details string      `json:"details,omitempty"`
-	UUID    UUID        `json:"uuid,omitempty"`
-	Rows    []ResultRow `json:"rows,omitempty"`
+	Count   *int         `json:"count,omitempty"`
+	Error   *string      `json:"error,omitempty"`
+	Details *string      `json:"details,omitempty"`
```

Comment:
There is no `Details` filed

Reply:
it was originally there and it is used in libovsdb/examples